src/collections/utils.ts

```diff
@@ -56,6 +56,19 @@ export const parseUri = (uri: string): ParsedUri => {
   };
 };
 
+function getBaseAPIPath(keyspaceName?: string, pathFromUrl?: string | null){
+  if(!pathFromUrl){
+    return '';
+  }
+  if(!keyspaceName){
+    return pathFromUrl;
+  }
+  const pathElements = pathFromUrl.split("/");
+  pathElements[pathElements.lastIndexOf(keyspaceName)] = '';
```

Comment:
Why not just `pathElements[pathElements - 1] = ''` ?

Reply:
Hi @vkarpov15 
I believe you meant `pathElements[pathElements.length - 1] = '';` that also works and thats even simpler. I pushed a commit [f5fc995](https://github.com/stargate/stargate-mongoose/pull/32/commits/f5fc995722eb3ddbefdf9b8cb22b47eceb47072b) with this change, merging this PR now.